pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -160,14 +164,13 @@ func (ks *scaler) handleScaleToZero(ctx context.Context, pa *pav1alpha1.PodAutos
 		// If we are stuck activating for longer than our progress deadline, presume we cannot succeed and scale to 0.
 		if pa.Status.CanFailActivation(now, activationTimeout) {
 			logger.Infof("Activation has timed out after %v.", activationTimeout)
-			return 0, true
+			return desiredScale, true
```

Comment:
Why not keep a straight `0` here to keep this very "in your face"?

Reply:
Everywhere else we're doing `desiredScale`, though it's obviously 0.
I can go in that direction, if you prefer.

Reply:
Oh I see, nevermind. We're in "handleScaleToZero" 😂